examples/ELASTIC_T/BORN_MATRIX/Silicon/elastic.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python -i
+# preceding line should have path for Python on your machine
+
+# elastic.py
+# Purpose: demonstrate elastic constant calculation for
+# two different crystal supercells, one with non-standard orientation
+#
+# Syntax:  elastic.py
+#          uses in.elastic as LAMMPS input script
+
+from __future__ import print_function
+import sys
+from elastic_utils import *
+
+np.set_printoptions(precision = 3, suppress=True)
+
+# setup MPI, if wanted
+me = 0
+# uncomment this if running in parallel via mpi4py
+from mpi4py import MPI
```

Comment:
This can be made more portable by using `lammps.extract_setting("world_rank")` and `lammps.extract_setting("world_size")` from the LAMMPS python module which will make loading mpi4py directly redundant and should also work in parallel without mpi4py for as long as no MPI communicator needs to be passed from python to LAMMPS. Please see:
https://docs.lammps.org/Python_module.html#lammps.lammps.extract_setting
https://docs.lammps.org/Library_properties.html#_CPPv422lammps_extract_settingPvPKc

Reply:
That's nice to know. I added it.